docs/InspectJs.md

```diff
@@ -0,0 +1,17 @@
+## Inspecting JS code
+
+1) Run js server with `--inspect` switch:
+```sh
+node --inspect ubuntu-server.js
+```
+
+2) Run your `react-native-desktop` application
+
+3) Open `chrome://inspect` page in a Chrome browser
+![](media/devtools-inspect-target.png)
+
+4) Click on `inspect` link near `ubuntu-server.js` Target
+
+5) Chrome devtools will open where you can see console output of your application and debug it.
+
+![](media/devtools-window.png)
```

Comment:
Maybe it's also worth to add a note about standalone React dev-tools: https://github.com/facebook/react-devtools/blob/master/packages/react-devtools/README.md. It allows to see React component hierarchy. Not sure if `Chrome://inspect` can show it.